src/App.js

```diff
@@ -48,13 +48,8 @@ function App() {
       setStatus('authenticated');
 
       // load stream
-      const streamJsonResponse = await fetchStream();
-      const stream = streamJsonResponse.data;
-
-      await store.dispatch({
-        type: 'stream/set',
-        stream: stream,
-      });
+      const responseJson = await fetchStreamGet();
+      await store.dispatch(responseJson.redux);
```

Comment:
new request convention on the front-end. every payload that we get from the server should be able to be directly plugged into dispatch.

a lot of existing requests do not follow this format, but slowly we should be transitioning these over.

here is the backend PR responsible for converting over this endpoint to the new format: https://github.com/0x213F/jukebox-radio-django/pull/17